src/app/portal/page.tsx

```
@@ -43,6 +44,7 @@ const Portal = () => {
       refetchOnWindowFocus: false,
     },
   );
+  console.log({ clerkUsers });
```

ESLint

[error] Unexpected console statement.

(no-console)